src/frontend/src/catalog/table_catalog.rs

```diff
@@ -107,7 +107,7 @@ pub struct TableCatalog {
     /// is not supported yet and expected to be `[0..columns.len()]`
     pub value_indices: Vec<usize>,
 
-    /// Definition of the materialized view.
+    /// The full `CREATE TABLE` or `CREATE MATERIALIZED VIEW` definition of the table.
```

Comment:
The `definition` is also used in system table `pg_matviews`, which was already used by the cloud team. I'm not sure how they are using it. We'd better check with them first.

Reply:
https://github.com/risingwavelabs/risingwave-cloud/blob/2c28ddead1cab0a298c314d9ab8a0119be520e81/mgmt/internal/model/mat_view.go#L238

Reply:
https://github.com/risingwavelabs/risingwave-cloud/pull/844

Looks like the definition is only used for representing to users in a highlight style. cc @Ad-Bean